js/app.jsx

```diff
@@ -486,6 +486,26 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      var lines = [];
+      if (str) {
+        lines = str.split(/,|ÿ/)
```

Comment:
https://github.com/twitter/scalding/blob/develop/scalding-core/src/main/scala/com/twitter/scalding/RichPipe.scala#L58-L62

Can we use `String.fromCharCode(255)`? Also, `/,|ÿ/` is equiv to `[,ÿ]`.